src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java

```diff
@@ -53,6 +53,9 @@
     @JsonProperty("HostsPath")
     private String hostsPath;
 
+    @JsonProperty("LogPath")
+    private String logPath;
```

Comment:
It's present since I remember. Even in your test inspectContainerResponse_full.json there is LogPath element